### PR TITLE
 Adding the get command, which allows download a specific file

### PR DESCRIPTION
**Pull request by lucasncabral**
This PR adds the implementation of the get command. This command allows the user to download an specific file from an entity. With it, when downloading, a copy of the file is created in the user's current directory, thus not being tracked by ml-git.
This command can be used whether or not in a directory with an initialized ml-git project.

```
$ ml-git datasets get --help
Usage: ml-git datasets get [OPTIONS] ML_ENTITY_NAME FILE_PATH

  Download a file tracked by ml-git into the current working directory.

Options:
  --config-repository TEXT  Define the configuration repository URL if it is
                            not in a directory with an initialized ml-git
                            project.
  --version TEXT            Define the entity version that should be used to
                            search for the desired file [default: latest].
  --wizard                  Enable the wizard to request information when
                            needed.
  --verbose                 Debug mode
```

**USAGE:**
```
$ ml-git datasets get dataset-ex path/to/my-file.png
INFO - Repository: Downloading files needed to mount the desired file
blobs: 100%|██████████████████████████████████████| 1.00/1.00 [00:01<00:00, 1.66s/blobs]
chunks: 100%|██████████████████████████████████████| 1.00/1.00 [00:01<00:00, 1.66s/chunks]
INFO - Repository: Mounting file [my-file.png] in current directory [D:\demo].
mounting file: 100%|██████████████████████████████████████| 1.00/1.00 [00:00<00:00, 62.5mounting file/s]
INFO - Repository: File mounted successfully.
```

**API:**
```
project = MLGitAPI()
project.get('datasets', 'dataset-ex', 'file.jpg', config_repository='user/config_repository')
```

### CODE REVIEW

**Review by MatheusClaudi — APPROVED** [review]

File: docs/mlgit_commands.md [hunk 1]
```diff
@@ -407,6 +407,44 @@ It will return the list of blobs that are corrupted/missing if the user passes t
...
+                            not in a directory with an initialized ml-git
+                            project.
+  --version TEXT            Define the entity version that should be used to
+                            search for the desired file [default: latest].
+  --wizard                  Enable the wizard to request information when
+                            needed.
+  --verbose                 Debug mode
+```
+
+This command allows the user to download an specific file from an entity. With it, when downloading, a copy of the file is created in the user's current directory, thus not being tracked by ml-git.
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:430]:
```suggestion
This command allows the user to download a specific file from an entity. With it, when downloading, a copy of the file is created in the user's current directory, with the file not being tracked by ml-git.
```

File: docs/mlgit_commands.md [hunk 2]
```diff
@@ -407,6 +407,44 @@ It will return the list of blobs that are corrupted/missing if the user passes t
...
+
+It is important to note that this command can be used whether or not in a directory with an initialized ml-git project.
+
+To download a file from within an initialized project:
+```
+ml-git datasets get dataset-ex data/my-file.png
+```
+
+To download the file without having to be in an ml-git project :
```
Comment (Vieira-Carlos) [inline docs/mlgit_commands.md:439]:
```suggestion
To download the file without having to be in an ml-git project:
```

**Review by Vieira-Carlos — COMMENTED** [review]
Just small changes

File: docs/mlgit_commands.md (hunk 1 above)
Comment (lucasncabral) [reply docs/mlgit_commands.md]: Suggestion applied at [38eb14b](https://github.com/HPInc/ml-git/pull/274/commits/38eb14bdf6224851cd2b473d2b4ec3d0aa4d7776).

**Review by Vieira-Carlos — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

**Pull request merged**